packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/constants.js

```diff
@@ -0,0 +1,21 @@
+export const TRACKING_SUFFIX = 'l7-policies';
+
+export const ACTIONS = {
+  REDIRECT_TO_URL: 'redirectToURL',
+  REDIRECT_TO_POOL: 'redirectToPool',
+  REDIRECT_PREFIX: 'redirectPrefix',
+  REJECT: 'reject',
+};
+
+export const ACTION_LABELS = {
+  [ACTIONS.REDIRECT_TO_URL]: 'Redirect to URL',
+  [ACTIONS.REDIRECT_TO_POOL]: 'Redirect to Pool',
+  [ACTIONS.REDIRECT_PREFIX]: 'Redirect Prefix',
+  [ACTIONS.REJECT]: 'Reject',
+};
```

Comment:
```suggestion
export const ACTIONS = {
  REDIRECT_TO_URL: {
   name: 'redirectToURL',
   label: 'Redirect to URL'
  },
  REDIRECT_TO_POOL: {
    name: 'redirectToPool',
    label: 'Redirect to Pool'
  },
  REDIRECT_PREFIX: {
    name: 'redirectPrefix',
    label: 'Redirect Prefix'
  },
  REJECT: {
    name: 'reject',
    label: 'Reject'
  }
};
```
Is it possible to merge both Actions and Labels ?